src/example.jsx

```diff
@@ -605,6 +606,19 @@ class ExampleApp extends React.PureComponent<Props, State> {
             )}
           </div>
         </article>
+        <article className="gc-panel gc-panel--separator">
+          <header className="gc-panel__title">Dropdown Menu</header>
+          <div className="gc-panel__content">
+            <Dropdown
+              button="-Select Value-"
+              label="Dropdown label"
+              items={[
+                { label: "Color", value: "Red" },
```

Comment:
`label` should be more something like `Red` when value is `red`, it should just represent human readable value.